assets/css/typingtest.css

```diff
@@ -0,0 +1,109 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Raleway&display=swap");
+
+* {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+  list-style-type: none;
+  text-decoration: none;
+  font-size: 10px;
+  font-family: "Poppins", sans-serif;
+}
+
+.container {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  background: linear-gradient(to right, #f4d03f, #16a085);
+  text-align: center;
+}
+
+.main {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+#heading {
+  font-size: 4rem;
+  margin-bottom: 50px;
+}
+
+#paragraph {
+  color: blue;
+  font-size: 2.5rem;
+  text-align: justify;
+  margin-bottom: 50px;
+  margin-top: 50px;
```

Comment:
![image](https://user-images.githubusercontent.com/74038190/221137754-61a0edba-94e8-4e53-bec3-fb0985d5ca51.png)

```suggestion
  margin-top: 150px;
```

add more margin to avoid overlapping of heading with text.